DOCUMENTATION.md

```diff
@@ -581,7 +581,7 @@ Hazelcast serializes all your objects before sending them to the server. Certain
 
 > **NOTE: The `Long` type means the type provided by [long.js library](https://github.com/dcodeIO/long.js).**
 
-> **NOTE: A `number` is serialized as `Double` by default. You can configure this behavior using the `serialization.defaultNumberType` config option.**
+> **NOTE: A `number` is serialized as `Double` by default. You can configure this behavior using the [serialization.defaultNumberType](http://hazelcast.github.io/hazelcast-nodejs-client/api/4.1.0/docs/interfaces/_config_serializationconfig_.serializationconfig.html#defaultnumbertype) config option.
```

Comment:
Is there a way to not use the exact version number in the URL(something like `latest` instead of `4.1.0`)? We might forget to update the link sometime later, after the future releases

Reply:
Also, the `**` is missing at the end of the sentence

Reply:
done thanks

Reply:
but looks full link does not work, this works `http://hazelcast.github.io/hazelcast-nodejs-client/api/current/docs`